Library/Homebrew/dev-cmd/test.rb

```diff
@@ -61,10 +61,11 @@ def test
         next
       end
 
-      # Don't test formulae missing test dependencies
+      # Don't test formulae missing test or fetch dependencies
```

Comment:
I actually think this can stay as-is, it's fine to test these.

Reply:
Done, I went back to the original version.